src/wrapper.jl

```diff
@@ -1051,4 +1203,151 @@ function FT_RestartInTask(ftHandle::FT_HANDLE)
+    FT_GetLatencyTimer(ftHandle::FT_HANDLE)
+
+Get the current latency timer in milliseconds.
+
+# Example
+
+See `FT_GetLatencyTimer`.
+"""
+function FT_GetLatencyTimer(ftHandle::FT_HANDLE)
+  pucTimer = Ref{UCHAR}()
+  status = ccall((:FT_GetLatencyTimer, libftd2xx), cdecl, FT_STATUS, 
+                 (FT_HANDLE, Ref{UCHAR}),
+                  ftHandle,  pucTimer)
+  check(status)
+  pucTimer[]
+end
+
+
+"""
+    FT_SetLatencyTimer(ftHandle::FT_HANDLE, ucTimer)
+
+Get the current latency timer value in milliseconds.
+
+# Example
+
+```julia-repl
+julia> numdevs = FT_CreateDeviceInfoList()
+0x00000004
+
+julia> handle = FT_Open(0)
+FT_HANDLE(Ptr{Nothing} @0x00000000051e56c0)
+
+julia> FT_SetLatencyTimer(handle, 0x0A)
+
+julia> FT_GetLatencyTimer(handle)
+0x0a
+
+See `FT_GetLatencyTimer`.
+"""
+function FT_SetLatencyTimer(ftHandle::FT_HANDLE, ucTimer)
+  status = ccall((:FT_SetLatencyTimer, libftd2xx), cdecl, FT_STATUS, 
+                 (FT_HANDLE, UCHAR),
+                  ftHandle,  ucTimer)
+  check(status)
+  return
+end
+
+
+"""
+    FT_GetBitMode(ftHandle::FT_HANDLE)
+
+Get the current bit mode.
+
+# Example
+
+See `FT_SetBitMode`.
+"""
+function FT_GetBitMode(ftHandle::FT_HANDLE)
+  pucMode = Ref{UCHAR}()
+  status = ccall((:FT_GetBitMode, libftd2xx), cdecl, FT_STATUS, 
+                 (FT_HANDLE, Ref{UCHAR}),
+                  ftHandle,  pucMode)
+  check(status)
+  pucMode[]
+end
+
+
+"""
+    FT_SetBitMode(ftHandle::FT_HANDLE, ucMask, ucMode)
+
+Enables different chip modes.
+
+`ucMask` sets up which bits are inputs and outputs. A bit value of 0 sets the corresponding
+pin to an input, a bit value of 1 sets the corresponding pin to an output. In the case of 
+CBUS Bit Bang, the upper nibble of this value controls which pins are inputs and outputs, 
+while the lower nibble controls which of the outputs are high and low.
+
+`ucModeMode` sets the chip mode, and must be one of the following: 
+0x0 = Reset 
+0x1 = Asynchronous Bit Bang 
+0x2 = MPSSE (FT2232, FT2232H, FT4232H and FT232H devices only) 
+0x4 = Synchronous Bit Bang (FT232R, FT245R,FT2232, FT2232H, FT4232H and FT232H devices only) 
+0x8 = MCU Host Bus Emulation Mode (FT2232, FT2232H, FT4232H and FT232H devices only) 
+0x10 = FastOpto-Isolated Serial Mode (FT2232, FT2232H, FT4232H and FT232H devices only) 
+0x20 = CBUS Bit Bang Mode (FT232Rand FT232H devices only) 
+0x40 = Single Channel Synchronous 245 FIFO Mode (FT2232H and FT232H devices only) 
+
+# Example
+
+```julia-repl
+julia> numdevs = FT_CreateDeviceInfoList()
+0x00000004
+
+julia> handle = FT_Open(0)
+FT_HANDLE(Ptr{Nothing} @0x00000000051e56c0)
+
+julia> FT_SetBitMode(handle, 0x0F, FT_MODE_ASYNC_BITBANG)
+
+julia> FT_GetBitMode(handle)
+0x01
+
+See [`FT_GetBitMode`](@ref).
+"""
+function FT_SetBitMode(ftHandle::FT_HANDLE, ucMask, ucMode)
+  @assert any(ucMode.==[FT_MODE_RESET,FT_MODE_ASYNC_BITBANG,FT_MODE_MPSSE,FT_MODE_SYNC_BITBANG,FT_MODE_MCU_EMULATION,FT_MODE_FAST_OPTO,FT_MODE_CBUS_BITBANG,FT_MODE_SCS_FIFO])
```

Comment:
As per above comment avoid potential array allocation and reduction by doing a set comparison with a tuple

Reply:
As above